modules/installation-guide/partials/con_persistent-volume-configuration.adoc

```diff
@@ -5,95 +5,123 @@
+For each user, all workspaces use the same Persistent Volume (PV) for the default data storage.
 
-.Restrictions on using the `common` PVC strategy
+When the user starts a first non-ephemeral workspace, the workspace engine creates the common persistent volume.
 
-When the `common` strategy is used, and a workspace PVC access mode is ReadWriteOnce (RWO), only one node can simultaneously use the PVC.
+When the user starts a non-ephemeral workspace, the workspace engine uses the common persistent volume.
+The workspace only binds simultaneously to one node in the {orch-name} cluster.
+The workspace engine ignores user-defined volumes. 
+The workspace engine replaces volumes related to user-defined volumes with a subpath in the common volume. 
+Subpaths have a `_<workspace-ID>_` or `__<original-volume-name>__` prefix.
+See xref:how-subpaths-are-used-in-persistent-volumes_{context}[].
+The {prod-short} Volume name is identical to the name of the user-defined persistent volume.
+Therefore, if a workspace container uses a {prod-short} volume with the same name as the user-defined persistent volume, they will use the same shared folder in the common persistent volume.
 
-If there are several nodes, you can use the `common` strategy, but:
+When the user deletes a workspace, the workspace engine deletes the corresponding subdirectory (`$\{ws-id}`) in the persistent volume directory.
 
-* The workspace PVC access mode must be reconfigured to `ReadWriteMany` (RWM), so multiple nodes can use this PVC simultaneously.
+When the user deletes the last workspace, the workspace engine removes the common persistent volume. 
 
-* Only one workspace in the same {orch-namespace} may be running. See xref:installation-guide:configuring-the-number-of-workspaces-that-a-user-can-run.adoc[].
+Restrictions on the `common` storage strategy with `ReadWriteOnce` access mode::
+The `ReadWriteOnce` access mode limits each user to run only one concurrent workspace.
+See xref:installation-guide:configuring-the-number-of-workspaces-that-a-user-can-run.adoc[].
 
-The `common` PVC strategy is not suitable for large multi-node clusters. Therefore, it is best to use it in single-node clusters. However, in combination with the `per-workspace` {orch-namespace} strategy, the `common` PVC strategy is usable for clusters with not more than 75 nodes. The PVC used with this strategy must be large enough to accommodate all projects to prevent a situation in which one project depletes the resources of others.
+Scalability::
+The `common` storage strategy is not suitable for multinode clusters with 'ReadWriteOnce' access mode for persistent volumes if the number of concurrently running workspaces per user is more than 1
 
-[id="the-per-workspace-pvc-strategy_{context}"]
-== The `per-workspace` PVC strategy
+Persistent volumes (PV) provisioning::
+Create large enough persistent volume to accommodate all projects to prevent a situation in which one project depletes the resources of others.
```

Comment:
```suggestion
Create a large enough PV to accommodate all projects to prevent a situation in which one project depletes the resources of others.
```